tutorials/Quick Start BLE.md

```diff
@@ -98,9 +98,8 @@ Requirements
 For Bluetooth LE you need a Bluetooth 4.0-capable adaptor in your computer (Bluetooth versions before 4.0 won't work). Pretty much all new computers come with Bluetooth 4, but you *may* need to get an external Bluetooth LE dongle if your computer:
 
 * Is an Apple Mac made before 2012
-* Is a Windows PC with a Windows version before 10
+* Is an older Windows or Linux device that predates Bluetooth 4.0
```

Comment:
I'm not sure this is a helpful change? How are people supposed to know if their device predates Bluetooth 4? Saying before windows 10 was accurate and easy to check.

Reply:
You can run Windows 10 on a device with Bluetooth 3.0

Perhaps we could add both? How about something like "a device older than Windows 10 or a device that doesn't support Bluetooth 4.0)